client/src/features/viewers/foh/Public.jsx

```diff
@@ -92,6 +121,7 @@ export default function Public(props) {
           isBackstage
           setCurrentPage={setCurrentPage}
           setPageNumber={setPageNumber}
+          format12={format12}
```

Comment:
Maybe instead of passing `format12` down as prop, and doing many `if` statements down the line, we could set up `settings.timeFormat` taking in account the query string too?

Reply:
I am not entirely sure what do you mean here, perhaps we could take a look on our next chat